example/24_batched_gemm_e_permute/batched_gemm_e_permute_xdl_fp16.cpp

```diff
@@ -1,253 +0,0 @@
-#include <iostream>
```

Comment:
batch GEMM + permute is no longer needed?

Reply:
Yes. We have another example cover this: https://github.com/ROCmSoftwarePlatform/composable_kernel/tree/examples_batched_grouped_gemm/example/25_gemm_bias_e_permute